app/diagnostics.go

```diff
@@ -602,6 +602,7 @@ func (a *App) trackConfig() {
 		"enable_uploads":                *cfg.PluginSettings.EnableUploads,
 		"allow_insecure_download_url":   *cfg.PluginSettings.AllowInsecureDownloadUrl,
 		"enable_health_check":           *cfg.PluginSettings.EnableHealthCheck,
+		"enable_marketplace":            *cfg.PluginSettings.EnableMarketplace,
```

Comment:
Can we add diagnostics to track when the marketplace is different from the default marketplace?

Reply:
@lieut-data any objections in always logging `MarketplaceUrl` instead?

Reply:
@ali-farooq0, actually, yes: we don't want to expose the internal details of a customer's configuration (e.g. the name of their internal host). When it's a free-form field, we just report when it's "different" to improve privacy.